Redacted: use a FL token when available

hypsometric opened the pull request:
Fixes #3163

doucheymcdoucherson commented:
not everyone is going to want to do this automatically - it needs to be configurable

matthewcky2k commented:
any update on this PR?

hypsometric commented:
@AdeHub 

- I just rebased against develop branch.
- The original PR had only the feature to use FL tokens implemented, I added the necessary configuration flags in params & UI to enable it, default is disabled.

AdeHub commented:
👍

topdollar3 commented:
Please can't you add this feature to orpheus?
